include/lm/lm_node.h

```diff
@@ -0,0 +1,52 @@
+/**
+ * @file lm_node.h
+ * @author Sean Massung
+ *
+ * All files in META are dual-licensed under the MIT and NCSA licenses. For more
+ * details, consult the file LICENSE.mit and LICENSE.ncsa in the root of the
+ * project.
+ */
+
+#ifndef META_LM_NODE_H_
+#define META_LM_NODE_H_
+
+#include <cstdint>
+#include <cstring>
+
+namespace meta
+{
+namespace lm
+{
+/**
+ * Simple struct to keep track of probabilities and backoff values that is
+ * packed into a uint64_t for storage.
+ */
+struct lm_node
```

Comment:
Is this class used externally anywhere, or is it an implementation detail?

If it's an implementation detail, maybe it's good to just tuck it away in the `.cpp` file where it's needed? Then I won't feel bad about it having no comments.